ui/src/app/richskill/service/rich-skill.service.ts

```diff
@@ -77,7 +77,7 @@ export class RichSkillService extends AbstractService {
     return this.httpClient
       .get(this.buildUrl(`${this.serviceUrl}/${uuid}`), {
         headers: this.wrapHeaders(new HttpHeaders({
-            Accept: "text/csv"
```

Comment:
I question that this is correct. The request here is for a CSV document. Maybe the endpoint is wrong? @jesusgudino ?

Reply:
I'm going to resolve this and move on. I hope this is correct.